ResearchKit/Common/ORKFormItemCell.m

```diff
@@ -776,6 +778,74 @@ - (BOOL)textView:(UITextView *)textView shouldChangeTextInRange:(NSRange)range r
 @end
 
 
+#pragma mark - ORKFormItemEligibilityCell
+
+@interface ORKFormItemEligibilityCell () <ORKEligibilitySelectionViewDelegate>
+
+@end
+
+
+@implementation ORKFormItemEligibilityCell {
+    ORKEligibilitySelectionView *_selectionView;
+}
+
+- (void)cellInit {
+    
+    // Add the selection view to the content view of the form item cell.
+    _selectionView = [[ORKEligibilitySelectionView alloc] initWithFrame:CGRectZero];
+    _selectionView.delegate = self;
+    [self.contentView addSubview:_selectionView];
+    
+    // Add the label to show the question.
+    ORKSubheadlineLabel *questionLabel = [ORKSubheadlineLabel new];
+    questionLabel.text = self.formItem.text;
+    questionLabel.numberOfLines = 0;
+    questionLabel.textAlignment = NSTextAlignmentCenter;
+    [self.contentView addSubview:questionLabel];
+    
+    self.contentView.layoutMargins = UIEdgeInsetsMake(kVMargin, kHMargin, kVMargin, kHMargin);
```

Comment:
The constants will need to be renamed to `HorizontalMargin` and `VerticalMargin` to merge with latest master.